src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -138,6 +138,12 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
              HelpMessage = "The list of request routing rule")]
         public PSApplicationGatewayRequestRoutingRule[] RequestRoutingRules { get; set; }
 
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "The list of RewriteRuleSets")]
+        public List<PSApplicationGatewayRewriteRuleSet> RewriteRuleSets { get; set; }
```

Comment:
@sai-manoj-kumar this should be renamed to `RewriteRuleSet`

Reply:
For this cmdlet, all other parameters are using plural name. So, to follow the current convention using the plural name.